### ISSUE TITLE
No cop checking for indentation of parameters?

### ISSUE THREAD

**Issue by bquorning** [bbatsov/rubocop#2173]
Checked on the latest master of RuboCop (0f260a67fd125697b0600f79f4457d5c17df0b70), this file produces no warnings:
```rb
# encoding: utf-8
def x(
    a: 1,
b: 2)
  puts a, b
end

def y(
    a,
b)
  puts a, b
end
```
The indentation is clearly wrong, but what would be the correct indentation?
```rb
def y(
      a,
      b
     )
  puts a, b
end

# or

def y(
  a,
  b
)
  puts a, b
end

# or

def y(a,
      b)
  puts a, b
end
```

**Comment by bquorning** [comment]
Ping @jonas054 :-)

**Comment by jonas054** [comment]
Thanks for the reminder. :smile: Well, first of all, they should of course be aligned. We should add that check to the duties of `Style/AlignParameters`.

About the parentheses, we're already checking in `Style/ClosingParenthesisIndentation` that "`)` [is indented] the same as the start of the line where `(` is" or that the parentheses are aligned if there's no line break inside the opening parenthesis. We could perhaps add some checking for symmetry. If one parenthesis (or curly brace or square bracket) has a line break on the inside, then the other one should have it too.
